src/components/button-toggle/button-toggle.scss

```diff
@@ -29,9 +29,44 @@
   }
 }
 
+.carbon-button-toggle--grouped {
+  .carbon-button-toggle__input:checked ~ .carbon-button-toggle__label{
+    border-color: $blue-sky;
+  }
+
+  + .carbon-button-toggle--grouped {
+    margin-left: -1px;
```

Comment:
does this need to not be applied to the first child?

Reply:
No, the intention is to squash two pixels of border into one throughout the internal elements,

The same effect could have been achieved with:

```
.all {
  border top, right and bottom 
  &:first-child {
    border left
  }
}
```

Reply:
ah yeah i missed the `+` 👍